doc/Whitepaper.md

```diff
@@ -828,21 +834,19 @@ these 64 bytes are hashed again with SHA-256 to make the symmetric key to be
 used for the session. It is also hashed a second time and the result copied over
 the first 8 bytes of the authentication header before the AuthType field is set.
 The effect being that the "Hash Code" field contains bytes 2 through 8 the hash
-of the hash of the password. This is used as a sort of username so that the
-other end knows which password to try using in the handshake.
-
-If Derivations is non-zero, an additional step is included, the two most
```

Comment:
Beware, Derivations has not gone away, Additional is effectively "Unused" though.